developer_workflow.md

```diff
@@ -1,41 +1,87 @@
+* When the entire card is complete, do a final review before moving to the team review process.
+* Push your branch to origin.
 * Submit a pull request on Github/BitBucket, and tag relevant reviewers from your team.
-* Push your branch to the integration environment after verifying no one else on the team is using it. Test the functionality of your feature on integration to make sure it works.
-* Move your card on the planning board to the "In Review" column.
+* Push your branch to the integration environment after verifying no one else on the team is using it. Verify that the functionality of your feature was properly deployed and works as expected.
+* Update your card's status to indicate it's ready for team review.
 
-### Pull Request Process
+# Team Review
 
 * Post in team chat that your pull request is ready for review and your branch is on the integration environment.
-* A team member other than the author reviews the pull request. Follow the (Code Review guidelines)[https://github.com/HaughtCodeworks/TeamGuide] to avoid miscommunication. Make comments and ask questions directly on lines of code in the GitHub/ BitBucket repo or in team chat.
-* Reviewers verify integration functionality before approving a pull request. Verification in integration should happen with a (QA mindset)[https://github.com/HaughtCodeworks/TeamGuide].
-* When satisfied, the reviewer approves and comments on the pull request “Ready to merge" or "+1”.
-* The author of the PR rebases interactively. Squash commits like “Fix whitespace” into one or a small number of valuable commit(s). Edit commit messages to reveal intent.
-* View a list of new commits and view changed files.
-* Run the test suite.
+* A team member other than the author reviews the pull request. Follow the [Code Review guidelines](https://github.com/HaughtCodeworks/TeamGuide) to avoid miscommunication. Make comments and ask questions directly on lines of code in the GitHub/ BitBucket repo or in team chat.
+* Reviewers verify integration functionality before approving a pull request. Verification in integration should happen with a [QA mindset](https://github.com/HaughtCodeworks/TeamGuide).
+* Reviewers comment on any issues they see or any improvements that they feel are important enough to address before approving the PR.
+* The author then addresses the feedback following the Development process above, introducing new commits as necessary.
+* Reviewers and the author continue this process of review and feedback until the PR can be approved. Approval can be a formal approval, +1 or a comment such as 'Ready to merge' or 'Looks good'.
+* Author of the PR reviews the branch's commits and determines any commits that need to be squashed through an interactive rebase.  Squash any meaningless commits such as the 'fixed whitespace' or 'missed this file' type so that all commits in the branch are valuable with meaningful messages.
```

Comment:
I'm thinking to tweak the end of this line to:

so that all commits in the branch are valuable on their own with meaningful messages.